apps/src/templates/instructions/InstructionsCSF.jsx

```diff
@@ -125,7 +125,7 @@ class InstructionsCSF extends React.Component {
    * again, we want to increase height.
    */
   UNSAFE_componentWillReceiveProps(nextProps) {
-    this.adjustRenderedHeight(nextProps);
+    this.debouncedAdjustRenderedHeight(nextProps);
```

Comment:
This is the only change that I'm unfamiliar with. I think this is part of the new resize logic that @breville added last year. Maybe this was just a single change that was missed as part of that work?

Reply:
If it's working well for you, and indeed improves things, then I can't think of a reason why there'd be a problem switching to the debounced version here.